tests/contrib/aioredis/test_aioredis.py

```diff
@@ -66,6 +68,19 @@ async def test_basic_request(redis_client):
     assert val is None
 
 
+@pytest.mark.asyncio
+@pytest.mark.snapshot
+async def test_closed_connection_pool(redis_client):
```

Comment:
I know most of the tests here don't have them... but would you be able to add a docstring explaining this test case.

Reply:
Yes, it was hard to understand what it is testing, added a docstring. Hope it is more clear now. Should I also add info where this is handled?